prevent uncaughtException

andrew0x0007c8 opened the pull request:
prevent uncaughtException

andrew0x0007c8 committed:
prevent uncaughtException

prevent uncaughtException

MarkTiedemann commented:
There is already an error handler in line 119 - it's just in the wrong place. Instead of only adding a second error handler in line 90, can you also remove the handler from line 119?

Otherwise, LGTM.

andrew0x0007c8 committed:
remove duplicate listener

I've put on error listener in the same level 'on connect' listener in case of situation where we got an error before connect (in that case process uncaught exception will occur). That case I've got while testing

andrew0x0007c8 commented:
I've put on error listener in the same level 'on connect' listener in case of situation where we got an error before connect (in that case process uncaught exception will occur). That case I've got while testing

MarkTiedemann committed:
Update index.js

MarkTiedemann commented:
This fix has been released as `v0.1.1`.

Thanks a lot for your PR, @andrew0x0007c8!
